### ISSUE TITLE
Property tab does not exist on type Chainable JQuery HTMLBodyElement

### ISSUE THREAD

**Issue by seanforyou23** [Bkucera/cypress-plugin-tab#19]
Hi there, thanks for this plugin! I'm trying to get it working for the first time and having a little trouble. I've installed the plugin, added `require('cypress-plugin-tab')` to cypress/support/index.js, but when I go to use the `tab()` method I receive the following error.

`Property 'tab' does not exist on type 'Chainable<JQuery<HTMLBodyElement>>'.ts(2339)`

<img width="780" alt="Screen Shot 2020-02-21 at 10 02 50 AM" src="https://user-images.githubusercontent.com/5942899/75045631-a268f500-5491-11ea-8668-f6cd070a2f15.png">

I'm using this withing a lerna monorepo, in case it helps diagnose. I have a branch you can checkout to run it locally if you like, it lives here https://github.com/seanforyou23/patternfly-react/tree/cypress-tab-support. You can run this locally with the following:

```bash
git clone git@github.com:seanforyou23/patternfly-react.git && cd patternfly-react/
yarn && yarn build:pf4 && yarn start:demo-app
# then, while "yarn start:demo-app" is running, in another console run the following
yarn start:cypress
```

I'm trying to use it for a simple case with button, you can see where I attempt here https://github.com/seanforyou23/patternfly-react/commit/af77075342fa2f864598eb4b8039eddcb0a9e86d#diff-ad8dcb99e87958a35e5815311202b43dR21

Please let me know if you spot anything I'm doing wrong, looking forward to using this plugin. Thanks!!!

**Comment by Bkucera** [comment]
That is a typescript typeerror, meaning the type definitions aren't getting picked up. You can verify this by adding `// @ts-ignore` above the line of code.

Does this actually prevent your spec from running or is it only a red-underline in the IDE?

**Comment by seanforyou23** [comment]
Ah good call, yes it was actually working under the covers and the IDE complaining was the only thing. I can ignore as you recommended for now. Thank you!

**Comment by Bkucera** [comment]
@seanforyou23 well it would be nice to know why the types weren't picked up. What editor? as a workaround you might be able to place at the top of the file:
```js
/// <reference types="cypress-plugin-tab" />
```

**Comment by isthar13** [comment]
Hi, I'm having the same issue. I'm using VisualCode Studio.
I tried to include type "cypress-plugin-tab" on tsconfig.ts:
```
{
	"compilerOptions": {
		"strict": true,
		"baseUrl": "../node_modules",
		"target": "es5",
		"lib": ["es5", "dom"],
		"types": ["cypress", "cypress-plugin-tab"]
	},
	"include": [
		"**/*.ts"
	]
}
```
And I tried to add the comment on top of my file:
```
/// <reference types="cypress-plugin-tab" />
describe('Test', () => {
	it('can go to step 2', () => {
		cy.visit('/');
		cy.get("[data-cy=accountNumberInput] input").click()
		cy.get("[data-cy=accountNumberInput] input").tab()
	})
})
```
With the comment, the `cypress-plugin-tab` is marked as error saying `Cannot find type definition file for 'cypress-plugin-tab'.ts`

**Comment by AshMcConnell** [comment]
I'm having the same issue, did you manage to get it working @isthar13 ?

Thanks!

**Comment by markgaze** [comment]
I've created the above PR to fix this but a workaround at the moment is to add `cypress-plugin-tab/src` to your `types` array in the `tsconfig.json` file and this will find the declaration file (if the PR gets merged you will have to remove the `/src` when a new package is released 🙂

**Comment by markgaze** [comment]
> @markgaze do you know what happened with #29? it seems merged but not released confused Cypress also doesn't seem to have fixed this issue [cypress-io/cypress#299](https://github.com/cypress-io/cypress/issues/299) - does anyone know anything about it? thinkingcowboy_hat_face

Unfortunately not, I'd totally forgotten about this until your reply but I just checked and we're still using the workaround I mentioned in my comment above... Hopefully there'll be a release at some point and we can remove that!
